ETC_en.tsv

```diff
@@ -5164,24 +5164,24 @@ ETC_20150317_005163	Please defeat Moyabruka!
 ETC_20150317_005164	Polluted Pond 1
 ETC_20150317_005165	The relax bomb slows down your movement {nl} when you approach Upent!
 ETC_20150317_005166	So you are still alive...
-ETC_20150317_005167	Your role is done.{nl}Die in silence!
+ETC_20150317_005167	You're done.{nl}Die in silence!
 ETC_20150317_005168	Goddess Shaule
 ETC_20150317_005169	Restraining Magic Circle
 ETC_20150317_005170	Temple Border
 ETC_20150317_005171	Goddess Shaule
 ETC_20150317_005172	Portal
-ETC_20150317_005173	Please defeat Golem!
+ETC_20150317_005173	Please defeat the Golem!
 ETC_20150317_005174	Please defeat Spanashon!
 ETC_20150317_005175	Please defeat Yekub!
 ETC_20150317_005176	Patient
 ETC_20150317_005177	Trigger to kill soldiers
-ETC_20150317_005178	Please defeat Yonazolem and obtain the easy burning stone!
```

Comment:
There is "please defeat" xxx everywhere . Seems to be an objective.
I guess it must be "Defeat" or "Please defeat" everywhere